source/unified-test-format/tests/valid-pass/poc-queryable-encryption.yml

```
@@ -0,0 +1,84 @@
+description: poc-queryable-encryption
+
+schemaVersion: "1.23"
+
+runOnRequirements:
+  - minServerVersion: "8.0"
```

```suggestion
  - minServerVersion: "7.0"
     csfle: true
```

Suggest using server 7.0 as the minimum to expand coverage. This test does not use `bulkWrite` (added in 8.0), and QE is Generally Available in server 7.0.

Use the `csfle` [requirement](https://github.com/mongodb/specifications/blob/a6897dc85129ddbb46ca540f4e10bca76aabae11/source/unified-test-format/unified-test-format.md#runonrequirement) to indicate libmongocrypt+(mongocryptd|crypt_shared) are required to run this test.